workspace/mastercard-apis-with-jwe-encryption-insomnia-workspace.json

```diff
@@ -0,0 +1,239 @@
+{
+    "_type": "export",
+    "__export_format": 4,
+    "__export_date": "2024-11-08T16:36:55.973Z",
+    "__export_source": "insomnia.desktop.app:v10.0.0",
+    "resources": [
+      {
+        "_id": "req_ef8350a9a0df4ca1a07c2af2202320b5",
+        "parentId": "wrk_c42f7dbb091f4d1fae4c91718ca39fc0",
+        "modified": 1572606757333,
+        "created": 1572009619195,
+        "url": "{{ host  }}/atms/v1/atm?PageOffset=0&PageLength=5&PostalCode=11101&Format=JSON",
+        "name": "Mastercard ATM Location (update me!)",
```

Comment:
Please rename it to Mastercard Service I want to use

Reply:
renamed